### PR TITLE
2.0 Installer Work On Linux Based Distros.

### PR DESCRIPTION
**Pull request by Cloud-121**
This is my attempt to get the installer working on linux based systems, List below is what is being worked on.

- [x] Adding start.sh and RUN_THIS_FIRST.sh
- [x] Make the installer.py actually run and install
- [x] Make the app run as root
- [x] [Install the deps for pywebview](https://github.com/r0x0r/pywebview/issues/805)
- [x] Have the helpers actually work on debian based + arch systems.
- [x] Don't have the folder only be root access 
- [x] Check if tkinker is install (Should be by default), if not install it.
- [x] Auto install dlls for Linux
- [ ] Auto install dlls for windows

**The app will not work till [#255](https://github.com/ETS2LA/Euro-Truck-Simulator-2-Lane-Assist/pull/255) in the Rewrite branch is merged.**

### CODE REVIEW

File: START_linux.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+echo "Activating virtual environment..."
+source venv/bin/activate
+
+echo "Checking requirements..."
+pip install -q -r code/requirements.txt
+echo "+ Done"
+
+echo "Running the installer / app..."
+sleep 1
```
Comment (Cloud-121) [inline START_linux.sh:28]: Remind myself to remove this

**Comment by Cloud-121** [comment]
After @DylDevs tests the automatic windows dll installer this PR is ready for review and to be merged.

**Comment by tazyooooazure** [comment]
container

**Comment by r4ygapist** [comment]
HOORAY

**Comment by Cloud-121** [comment]
> https://github.com/r0x0r/pywebview/issues/805 is that issue solved yet?

Wasn't even the issue xd, Turned out to be something else.

**Pull request merged**